### PR TITLE
Added a robotic favicon and sleek background for the webserver

### PR DESCRIPTION
**Pull request by AlphaT3ch**

### CODE REVIEW

**Comment by botwillacceptanything** [comment]
#### :ballot_box_with_check: Voting procedure reminder:
To cast a vote, post a comment containing `:+1:` (:+1:), or `:-1:` (:-1:).
Remember, you **must :star:star this repo for your vote to count.**

All comments within this discussion are searched for votes, regardless of the time of posting.
You can cast as many votes as you want, but only the last one will be counted.
(You may consider editing your comment instead of adding a new one.)
Comments containing both up- and down-votes are disregarded.
PR authors automatically count as a :+1: vote.

A decision will be made after this PR has been open for **15** minutes (plus/minus **10** percent, to avoid people timing their votes), and at least **8** votes have been made.
A supermajority of **65%** is required for the vote to pass.

*NOTE: the PR will be closed if any new commits are added after:* 05b828ed8e28d2033724fa73b8823b222fba9a14

**Comment by dbpokorny** [comment]
:+1:

File: data/views/layouts/main.hbs [hunk 1]
```diff
@@ -5,13 +5,15 @@
...
-        <style type="text/css" media="all">body {
+        <style type="text/css" media="all">
+	    body {
             margin: 0px;
             padding: 0px;
             background-repeat: no-repeat;
             background-position: top right;
             background-color: white;
             color: black;
+	    background: url('/background.jpg?raw=true');
```
Comment (jedahan) [inline data/views/layouts/main.hbs]: MISSING SPACE WAAAH :-p

**Comment by RyanMilb** [comment]
:+1:

**Comment by avdg** [comment]
:-1: Can't have one size, fits all image background picture

**Comment by AlphaT3ch** [comment]
@avdg: I don't understand
@jedahan: Whoops... I'm sure we'll survive though ;)

**Comment by avdg** [comment]
Keep in mind that there are different resolutions, though full hd (1980*1080) is pretty common these days there are older smaller screens, mobile devices, bigger screens, etc...

**Comment by LifeMushroom** [comment]
:+1:

**Pull request closed without merging**